### PR TITLE
Individual items can now be disabled.

### PR DESCRIPTION
**Pull request by DsosaV**
Solves #157

### CODE REVIEW

**Comment by DsosaV** [comment]
isDisabled property is now an optional property for the ListItem class.

All test cases pass. Note that I added one test case for disabling individual items.

File: src/ng-multiselect-dropdown/test/multi-select.component.spec.ts [hunk 1]
```diff
@@ -12,13 +12,13 @@ class Ng2MultiSelectDropdownSingleSelect {
   @ViewChild(MultiSelectComponent)
   select: MultiSelectComponent;
   cities = [
-    { item_id: 1, item_text: 'Mumbai' },
-    { item_id: 2, item_text: 'Bangalore' },
-    { item_id: 3, item_text: 'Pune' },
-    { item_id: 4, item_text: 'Navsari' },
-    { item_id: 5, item_text: 'New Delhi' }
+    { item_id: 1, item_text: 'Mumbai', isDisabled: false },
```
Comment (NileshPatel17) [inline src/ng-multiselect-dropdown/test/multi-select.component.spec.ts]: Please undo these changes. It should work without isDisabled property as it is optional now.

Comment (NileshPatel17) [reply src/ng-multiselect-dropdown/test/multi-select.component.spec.ts]: You may have one item with isDisabled true,and have respective test case.

File: src/ng-multiselect-dropdown/test/multi-select.component.spec.ts [hunk 2]
```diff
@@ -157,6 +157,20 @@ describe('ng-multiselect-component', function() {
       tickAndDetectChanges(fixture);
       expect(fixture.componentInstance.select.isDropdownOpen).toBe(false);
     }));
+
+    it('should not select disabled item', fakeAsync(() => {
```
Comment (NileshPatel17) [inline src/ng-multiselect-dropdown/test/multi-select.component.spec.ts]: how this test case is getting passed. You do not have a single item with isDisabled true.

Comment (NileshPatel17) [reply src/ng-multiselect-dropdown/test/multi-select.component.spec.ts]: okay, you have explicitly disabled it(line no 163). my bad :)

Comment (DsosaV) [reply src/ng-multiselect-dropdown/test/multi-select.component.spec.ts]: There's an issue with type in line 163 if no "isDisabled" property is defined for the elements in the cities array, since it is of implicit type. To delete all those isDisabled properties, I would have to define a type for those values, declare the array with that explicit type and then omit those properties.

something around the works of:
type Item = {
  item_id: number,
  item_text: string,
  isDisabled?: boolean
};

and then:

cities: Item[] = [
    { item_id: 1, item_text: 'Mumbai' },
    { item_id: 2, item_text: 'Bangalore' },
    { item_id: 3, item_text: 'Pune' },
    { item_id: 4, item_text: 'Navsari' },
    { item_id: 5, item_text: 'New Delhi' }
];

Comment (DsosaV) [reply src/ng-multiselect-dropdown/test/multi-select.component.spec.ts]: I just didn't want to declare a new type for the test and assumed adding the property to each element was easier.

Comment (DsosaV) [reply src/ng-multiselect-dropdown/test/multi-select.component.spec.ts]: > 
> 
> okay, you have explicitly disabled it(line no 163). my bad :)
> I would suggest you to have a item with true flag, do not disable it explicitly. this way, sample code will showcase how disabled flag behaves.

So, do you want me to declare a type for those items and set one to true? Or should I leave those isDisabled properties there and just set one to true?

Comment (DsosaV) [reply src/ng-multiselect-dropdown/test/multi-select.component.spec.ts]: Also, want me to add changes to the demo so it shows this new feature?

File: src/ng-multiselect-dropdown/test/multi-select.component.spec.ts (hunk 1 above)
Comment (DsosaV) [reply src/ng-multiselect-dropdown/test/multi-select.component.spec.ts]: Done.

File: src/ng-multiselect-dropdown/test/multi-select.component.spec.ts (hunk 2 above)
Comment (NileshPatel17) [reply src/ng-multiselect-dropdown/test/multi-select.component.spec.ts]: please add to the demo. is it good practice to merge PR with 1 commit. can you please rebase your all commit?

Comment (DsosaV) [reply src/ng-multiselect-dropdown/test/multi-select.component.spec.ts]: I added the feature to the multiple-demo and also squashed all commits into one.

**Comment by DsosaV** [comment]
@NileshPatel17 Is there anything you want me to do before merging this PR?

**Comment by Sam2626** [comment]
hello, Can you please merge this pull request as early as possible because I need this functionality , It
's very urgent for me so please understand my concern.
Thanks In Advance
